### ISSUE TITLE
Non TLS deployments should still use the domain for the ingress.

### ISSUE THREAD

**Issue by KaelBaldwin** [devspace-cloud/devspace-cloud#13]
running:
`devspace run deploy-devspace-cloud-no-tls`

Currently sets `--var DOMAIN=""` and I'm not sure why, but it's causing the ingress to have no host value.

I'll create a PR to fix this momentarily.

**Comment by FabianKramm** [comment]
@KaelBaldwin thanks for opening this issue! We omit the domain name when deploying without tls so that devspace cloud accepts incoming connections from all hosts (*). If you want to restrict devspace cloud to only listen on a special domain you can run `devspace run deploy-devspace-cloud-no-tls --var DOMAIN=my-domain.com`

**Comment by KaelBaldwin** [comment]
OK, I see what you are saying.

I could see how this would be good on a cluster devoted to devspace, but in our cluster, I'm using devspace alongside other deployments with their own ingresses and domains directly pointing to them.

My expectation in a no-tls option was that it would just disable the tls aspect, not ignore the domain that I had explicitly defined earlier via the prompt at run time.

I'm not 100% sure on the priority of nginx-ingress. If a path is specified and no host, will it take priority over an ingress that has a host defined but just a '/' path? If it does, this could cause traffic expected to go to the explicit domain to go to devspace-cloud instead.

**Comment by LukasGentele** [comment]
@KaelBaldwin I agree: the current solution is not very intuitive and having an ingress without a hostname is indeed not ideal if there is anything else running within the same cluster. I believe @FabianKramm wanted to have a good user experience for users that are evaluating DevSpace Cloud in local cluster like minikube. Maybe we should rename `deploy-devspace-cloud-no-tls` to `deploy-devspace-cloud-local` and additionally add a variable for TLS true/false that is set to false automatically for `deploy-devspace-cloud-local` and will be a question during the regular `deploy-devspace-cloud0`?

If I see a thumbs up from both of you, I will restructure this and update the README accordingly. If any of you see a problem with my proposed solution, let me know.
